vendor/vendor.json

```diff
@@ -163,7 +163,7 @@
 		{"path":"github.com/dustin/go-humanize","checksumSHA1":"xteP9Px90oMrg/HZuqvZkpXCR+s=","revision":"8929fe90cee4b2cb9deb468b51fb34eba64d1bf0"},
 		{"path":"github.com/elazarl/go-bindata-assetfs","checksumSHA1":"7DxViusFRJ7UPH0jZqYatwDrOkY=","revision":"30f82fa23fd844bd5bb1e5f216db87fd77b5eb43","revisionTime":"2017-02-27T21:27:28Z"},
 		{"path":"github.com/fatih/color","checksumSHA1":"VsE3zx2d8kpwj97TWhYddzAwBrY=","revision":"507f6050b8568533fb3f5504de8e5205fa62a114","revisionTime":"2018-02-13T13:34:03Z"},
-		{"path":"github.com/fsouza/go-dockerclient","checksumSHA1":"ep1kI8mdel8vXtOCdVlaLmyvxlA=","revision":"97b4aba9b2565d0313a8a9701626e47b3ef8c490","revisionTime":"2020-02-20T19:25:13Z","version":"v1.6.3","versionExact":"v1.6.3"},
+		{"path":"github.com/fsouza/go-dockerclient","checksumSHA1":"hrVrIULvQbbKeY52P+ynQ/GiXZ0=","revision":"9abb9ed850fd14325e1f2c7d4fa47960baf980eb","revisionTime":"2020-04-23T12:37:09Z","version":"v1.6.5","versionExact":"v1.6.5"},
 		{"path":"github.com/fsouza/go-dockerclient/internal/archive","checksumSHA1":"YJ7WR4AVtD2ykYJr+1mTtazkma0=","revision":"01c3e9bd8551d675a60382c0303ef51f5849ea99","revisionTime":"2018-11-29T02:57:25Z","version":"v1.6.3","versionExact":"v1.6.3"},
```

Comment:
Good catch!  Thank you so much for fixing this.  Mind if you update the rest internal subpackages so all point to the same version?  Also, we'd appreciate you signing the CLA?

Reply:
Thanks for your reminder.

I have signed the CLA and remove the unused internal subpackages.